.github/workflows/build.yml

```diff
@@ -86,16 +94,22 @@ jobs:
           name: tests-result
           path: ${{ github.workspace }}/build/reports/tests
 
+      # Upload Kover report to CodeCov
+      - name: Upload Code Coverage Report
+        uses: codecov/codecov-action@v3
+        with:
+          files: ${{ github.workspace }}/build/reports/kover/xml/report.xml
```

Comment:
This was from upstream - hope it is ok to upload our code coverage reports to this service.